concrete-core/src/backends/core/implementation/engines/lwe_ciphertext_trivial_decryption.rs

```diff
@@ -0,0 +1,95 @@
+use crate::backends::core::private::crypto::encoding::Plaintext as ImplPlaintext;
+use crate::prelude::{
+    CoreEngine, LweCiphertext32, LweCiphertext64, LweCiphertextTrivialDecryptionEngine,
+    LweCiphertextTrivialDecryptionError, Plaintext32, Plaintext64,
+};
+
+impl LweCiphertextTrivialDecryptionEngine<LweCiphertext32, Plaintext32> for CoreEngine {
+    /// # Example:
+    ///
+    /// ```
+    /// # fn main() -> Result<(), Box<dyn std::error::Error>> {
+    ///
+    /// use concrete_commons::dispersion::Variance;
+    /// use concrete_commons::parameters::LweSize;
+    /// use concrete_core::prelude::*;
+    ///
+    /// // DISCLAIMER: the parameters used here are only for test purpose, and are not secure.
+    /// let lwe_size = LweSize(10);
+    /// let input = 3_u32 << 20;
+    ///
+    /// let mut engine = CoreEngine::new()?;
+    /// let plaintext: Plaintext32 = engine.create_plaintext(&input)?;
+    /// // DISCLAIMER: trivial encryption is NOT secure, and DOES NOT hide the message at all.
```

Comment:
Can I ask you to add this disclaimer to the lwe_ciphertext_trivial_encryption too please? (for u32 & u64). Thx